File: Week1/O_Case_Checker.cpp
#include<iostream>
#include<string>
#include <cctype>

using namespace std;

int main() {
    int n;
    cin >> n;
    string s;
    cin >> s;
    
    for (int i = 0; i < n; i++) {
        if (isupper(s[i])) {
            cout << "is upper";
        } else if (islower(s[i])) {
            cout << "is lower";
        } else if (isdigit(s[i])) {
            cout << "is digit";
        }
        
        cout << endl;
    }

    return 0;
}

File: Week1/O_Case_Checker.py
n=int(input())
k=input()

#if if the elment of k is digit or upper or lower case
for i in range(n):
    if k[i].isupper():
        print("is upper")
    elif k[i].islower():
        print("is lower")
    elif k[i].isdigit():
        print("is digit")
